dvc_ssh/__init__.py

```diff
@@ -3,22 +3,27 @@
 import threading
 from typing import ClassVar
 
-from funcy import memoize, silent, wrap_prop, wrap_with
+from funcy import memoize, wrap_prop, wrap_with
 
 from dvc.utils.objects import cached_property
 from dvc_objects.fs.base import FileSystem
 from dvc_objects.fs.utils import as_atomic
 
-DEFAULT_PORT = 22
-
 
 @wrap_with(threading.Lock())
 @memoize
 def ask_password(host, user, port, desc):
+    prompt = f"Enter a {desc} for"
+    if user:
+        prompt += f" {user}@{host}"
+    else:
+        prompt += f" {host}"
+    if port:
+        prompt += f":{port}"
+    prompt += ":\n"
```

Comment:
I had to make these conditional, since we don't use the defaults at the moment.

These will likely get removed when we start using a proper callback (will do in next PR).